Composer/packages/client/src/pages/design/exportSkillModal/content/Description.tsx

```diff
@@ -80,7 +81,9 @@ export const Description: React.FC<ContentProps> = ({ errors, value, schema, onC
   );
 
   useEffect(() => {
-    onChange({ $id: botName, name: botName, ...value });
+    if (!value.$id) {
+      onChange({ $schema, $id: botName, name: botName, ...rest });
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify. So looks like stringify cannot guarantee order when run on structures that dont maintain order. Wonder if u want to sort the keys to make sure schema always comes up first or we let PVA do it.

Reply:
@srinaath PVA is taking a bug to not enforce this constraint. I think we're OK here.